.github/workflows/integration.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: |
           cd $(go env GOPATH)/src/github.com/cri-o/cri-o
           make all test-binaries
-          # skip seccomp tests because they have permission denied issues in a container
-          sudo -E test/test_runner.sh $(ls test/ | grep bats | grep -v seccomp)
+          # skip seccomp tests because they have permission denied issues in a container and accept signed image as they don't use conmon
+          sudo -E test/test_runner.sh $(ls test/ | grep bats | egrep -v seccomp\|image)
```

Comment:
Maybe replace `egrep` with `grep -E`?
See for example
* https://github.com/containers/podman/pull/18440
and https://unix.stackexchange.com/a/383454/9360

Reply:
good idea, fixed now